public-ui/src/app/shared/components/breadcrumb/breadcrumb-react.component.tsx

```diff
@@ -0,0 +1,68 @@
+
+import { Component, Input } from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { reactStyles } from 'app/utils';
+import { NavStore } from 'app/utils/navigation';
+import { environment } from 'environments/environment';
+import * as React from 'react';
+
+const styles = reactStyles({
+    preCrumb: {
+        color: '#0079b8'
+    },
+    crumb: {
+        color: '#2b266d'
+    }
+
+});
+
+const css = `
+    .crumb-container *{
+        margin-right: 0.5em;
+    }
+    .crumb-container {
+        margin-top:1em;
+        display:block;
+    }
+`;
```

Comment:
I think we should move this into the reactStyles object and drop the `<style>` tag from the template. It's better to have a single style method, even if we duplicate `marginRight: '0.5em'` a few times. Something like the suggested change above should work.